integrated_channels/degreed/migrations/0009_auto_20210119_1546.py

```diff
@@ -0,0 +1,73 @@
+        ),
+        migrations.AlterField(
+            model_name='degreedenterprisecustomerconfiguration',
+            name='key',
+            field=models.CharField(help_text='The API Client ID provided to edX by the enterprise customer to be used to make API calls to Degreed on behalf of the customer.', max_length=255, verbose_name='API Client ID'),
+        ),
+        migrations.AlterField(
+            model_name='degreedenterprisecustomerconfiguration',
+            name='secret',
+            field=models.CharField(help_text='The API Client Secret provided to edX by the enterprise customer to be used to make API calls to Degreed on behalf of the customer.', max_length=255, verbose_name='API Client Secret'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='degreed_base_url',
+            field=models.CharField(help_text='The base URL used for API requests to Degreed, i.e. https://degreed.com.', max_length=255, verbose_name='Degreed Base URL'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='degreed_company_id',
+            field=models.CharField(help_text='The organization code provided to the enterprise customer by Degreed.', max_length=255, verbose_name='Degreed Organization Code'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='degreed_user_id',
+            field=models.CharField(help_text='The Degreed User ID provided to the content provider by Degreed. It is required for getting the OAuth access token.', max_length=255, verbose_name='Degreed User ID'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='degreed_user_password',
+            field=models.CharField(help_text='The Degreed User Password provided to the content provider by Degreed. It is required for getting the OAuth access token.', max_length=255, verbose_name='Degreed User Password'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='key',
+            field=models.CharField(help_text='The API Client ID provided to edX by the enterprise customer to be used to make API calls to Degreed on behalf of the customer.', max_length=255, verbose_name='API Client ID'),
+        ),
+        migrations.AlterField(
+            model_name='historicaldegreedenterprisecustomerconfiguration',
+            name='secret',
+            field=models.CharField(help_text='The API Client Secret provided to edX by the enterprise customer to be used to make API calls to Degreed on behalf of the customer.', max_length=255, verbose_name='API Client Secret'),
```

Comment:
part of me is thinking "let's make the field less of a scroll-y boy", but also ehhh.

Reply:
do you mean the length of the line? I can happily move it to multiple lines.

Reply:
done!